capstone_project/src/main/webapp/pages/mapsTemplate.html

```diff
@@ -6,6 +6,8 @@
     <link rel="stylesheet" href="../styles/maps_style.css">
     <link href="https://fonts.googleapis.com/css?family=Quicksand:100,200,300" rel="stylesheet">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```

Comment:
If you're not using bootstrap, could you remove the css and js cdn

Reply:
I'm actually using bootstrap to create the tabs for the infowindows on the map!

Reply:
Oh okay, just be careful. They might have overlapping class declarations